gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
I think you (or whoever manages the ci builds) need to update the gradle version in Jenkins config, as `2.14.1` is the minimum gradle version required for the current android gradle plugin.

We can down-date the gradle version to `2.12`, but then we also need to down-date the android gradle plugin to below `2.1.2`. That will not be a problem right now, but instant run and other debugging features will not work, and it's better if we keep it updated as it'll keep increasing.

But if increasing the gradle version is not possible, let me know, and I'll send a PR to fix both versions.

For more info:

![image](https://cloud.githubusercontent.com/assets/6936968/23862719/c1ec6f7c-0833-11e7-97cf-8d15b8c1237e.png)
*Source* - https://developer.android.com/studio/releases/gradle-plugin.html